tools/update_extern.py

```diff
@@ -0,0 +1,169 @@
+#!/bin/env python
+
+"""
+This script is used to update the files in `sunpy/extern` while making new releases.
+"""
+
+import os
+import shutil
+from zipfile import ZipFile
+import requests
+from pathlib import Path
+
+# get parent directory of sunpy
+SUNPY_DIR = Path(__file__).parent.parent
+
+
+# List of packages to update
+PACKAGES = ["appdirs", "distro", "inflect", "parse"]
+
+# Dictionary to store the authors of the packages
+AUTHORS = {
+    "appdirs": "ActiveState",
+    "distro": "python-distro",
+    "inflect": "jaraco",
+    "parse": "r1chardj0n3s",
+}
```

Comment:
Does this need to be seperate?

Reply:
I am not sure, but this makes accessing authors easy otherwise we have to hardcode the author's name.

Reply:
Rather than having to write the package names twice, you can access them using the keys of the authors dictionary. My thinking is making sure it's easy to update this script when we add a new package to the extern directory.

Reply:
Sure, I will do that.